webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -29,6 +31,7 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
     && task.state === RunState.Paused;
   const isCancelable = isExperiment
     && cancellableRunStates.includes(task.state as RunState);
+  const canLaunchTensorboard = isExperiment;
```

Comment:
question: are there certain experiment states that can't be viewed in tensorboard? like `Pulling`, `Deleted`? If an experiment errors out, can it be viewed still under tensorboard?

Reply:
the only case we can be certain tensorboard cannot be viewed is: if the experiment has no data and we know for sure it will not have data in the future. from the list views (dashboard and experiment list) we don't have enough information to assert this situation but we can  do that in experiment detail by searching all the trials (if available)

Reply:
@sidneyw might know more here.

Reply:
The only time it for sure will not work is if the trial is in an error state and hasn't completed a step